### PR TITLE
[gem] Update mimemagic because of license

### PR DESCRIPTION
**Pull request by hallelujah**
The gem version has been yanked because of some licensing issues

https://github.com/mimemagicrb/mimemagic/issues/98

### CODE REVIEW

File: Gemfile.base [hunk 1]
```diff
@@ -91,7 +91,8 @@ group :development, :test do
   end
 end
 
-gem 'nokogiri', '>= 1.6.8'
+gem 'mimemagic', '~> 0.3.10'
+gem 'nokogiri', '~> 1.10.10'
```
Comment (gsaslis) [inline Gemfile.base:95]: I see we're not only updating the version of nokogiri (looking at the mimemagic linked PR some kind of version update does seem to be required though I was not able to spot exact versions recommended there), but we're also changing the policy here when choosing which nokogiri versions we can accept updates for: 
`>=` vs. `~>` 

is that intentional, or perhaps some copy-paste error? 🤔

Comment (hallelujah) [reply Gemfile.base:95]: This is correct, nokogiri 1.11+ drops support of ruby 2.4 which is used in some deployments (not disclosing which ones for security reasons 😖 ...)

Comment (gsaslis) [reply Gemfile.base:95]: Do you think this is worth also documenting in the commit message?

As long as we have it documented at least in this comment, I'm good with that.

**Review by guicassolato — APPROVED** [review]

File: Gemfile.base (hunk 1 above)
Comment (hallelujah) [reply Gemfile.base:95]: It is documented in another PR on the main branch https://github.com/3scale/porta/pull/2391

**Pull request merged**